src/mod_zone_difficulty_scripts.cpp

```diff
@@ -1184,11 +1218,11 @@ class mod_zone_difficulty_dungeonmaster : public CreatureScript
     {
         //LOG_INFO("module", "MOD-ZONE-DIFFICULTY: OnGossipHelloChromie");
         Group* group = player->GetGroup();
-        if (group && group->IsLfgRandomInstance() && !player->GetMap()->IsRaid())
-        {
-            creature->Whisper("I am sorry, time-traveler. You can not accept challenges here. You need to choose a specific dungeon in order to play my history lessons.", LANG_UNIVERSAL, player);
-            return true;
-        }
+        //if (group && group->IsLfgRandomInstance() && !player->GetMap()->IsRaid()) // Qeme
```

Comment:
This can't be just removed without a replacement. Feel free to add a config option.

Reply:
Added to config